tests/crypto.rs

```diff
@@ -0,0 +1,16 @@
+use telio_crypto::{PresharedKey, SecretKey};
+
+const SK: SecretKey = SecretKey::new([0xBAu8; 32]);
+const PRK: PresharedKey = PresharedKey::new([0xBAu8; 32]);
+
+#[ignore]
+#[test]
+fn secret_key_printing() {
+    // Not marking this whole test as only for release mode, since this can lead to us
+    // running it in debug and getting no failure. This way if you see this test run on
+    // CI (and pass), you know that it was run in the release mode.
+    #[cfg(debug_assertions)]
+    panic!("This test makes no sense in the debug mode, only run it in release mode");
+    assert_eq!("SecretKey(****)", format!("{SK:?}"));
+    assert_eq!("PresharedKey(****)", format!("{PRK:?}"));
```

Comment:
Very unimportant nit: Should the short version of `PresharedKey` be `PSK` instead of `PRK`?